### DISCUSSION TITLE
add links to issues and (this) discussion board

### DISCUSSION THREAD

**Discussion by jdoubleu** [plausible/analytics#1575] [Anything Else]
I was confused where to create a ticket for a feature request. The common way to go is to create a new issue. Luckily, I had to tick the message box, that my issue/bug isn't a feature request. However, it wasn't so straightforward where I should direct my feature request at. 

I suggest you add links to the `README.md` and [`CONTRIBUTING.md`](https://github.com/plausible/analytics/blob/master/CONTRIBUTING.md) guide directing new users to the correct place where to file an issue or suggest a new feature.

**Comment by metmarkosaric** [comment]
thanks @jdoubleu! we do link to this section in the issue template but i'll see if there are better places we can put it.
